### PR TITLE
[Enhancement] Support all new version Statement by new Parser

### LINKED ISSUE #6070: [Enhancement] Support all new version Statement by new Parser

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] others

## Which issues of this PR fixes ：

Fixes #6070 #4049

### CODE REVIEW

**Comment by HangyuanLiu** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

File: fe/fe-core/src/main/java/com/starrocks/analysis/DropWorkGroupStmt.java [hunk 1]
```diff
@@ -3,10 +3,8 @@
 
 import com.starrocks.sql.ast.AstVisitor;
 
-// Drop WorkGroup specified by name
```
Comment (mofeiatwork) [inline fe/fe-core/src/main/java/com/starrocks/analysis/DropWorkGroupStmt.java:6]: Why remove the comment?

**Review by murphyatwork — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/analysis/DropWorkGroupStmt.java (hunk 1 above)
Comment (HangyuanLiu) [reply fe/fe-core/src/main/java/com/starrocks/analysis/DropWorkGroupStmt.java:6]: I don't think this comment is meaningful. If I think it needs to be kept, then I will restore it.

**Review by kangkaisen — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 77 / 83 (92.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|75|81|92.59%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|1|100.00%|

**Pull request merged**